### ISSUE TITLE
Gatsby Develop fails

### ISSUE THREAD

**Issue by joseph-allen** [greglobinski/gatsby-starter-hero-blog#46]
after running
```
gatsby develop
```

I receive the following output:
```
error There was an error compiling the html.js component for the development server.

  Error: Module build failed: BrowserslistError: Unknown browser query `dead` (While processing preset: "/node_modules/babel-preset- env/lib/index.js")
```

**Comment by joseph-allen** [comment]
running

```
./node_modules/gatsby-cli/lib/index.js develop
```

works, I am guessing the version of gatsby I installed doesn't work with this.
my global install is v1.1.58
the one within this project is v2.4.1

I am guessing this is related to the other issue open at the moment so I will close :)

**Comment by greglobinski** [comment]
@joseph-allen 
I guess it's vice versa 
>my global install is v1.1.58
the one within this project is v2.4.1

**Comment by greglobinski** [comment]
This is a starter for Gatsby v1, it seems that it has problem with `gatsby-cli` v2

As @joseph-allen pointed the workaround is to use loclal `gatsby-cli`.

The convenient way is to update `package.json`

```
  "scripts": {
    "build": "./node_modules/.bin/gatsby build",
    "develop": "./node_modules/.bin/gatsby develop",
   ...
```

and instead `gatsby develop` and `gatsby build` use `yarn develop` and `yarn build`

**Comment by mohsenkhanpour** [comment]
Yup, I can confirm after updating the global cli, it couldn't build anymore. Works fine with npx though.
